### PR TITLE
change sos acl list display

### PR DESCRIPTION
**Pull request by illi-j**
This PR changes the output of the command `exo sos show` & `exo sos list acl`

BEFORE:
``` 
%exo sos acl list bucket-test object-test.pdf
┼───────────────────────┼──────────────┼──────────────────────────────────────┼
│       FILE NAME       │     ACL      │                VALUE                 │
┼───────────────────────┼──────────────┼──────────────────────────────────────┼
│ object-test.pdf       │ Write        │ 9f0250-98c4-41ec-a9a2-680f47ebdc     │
│ object-test.pdf       │ Full Control │ 1806e5-4013-4538-8d94-dde0e11e31     │
┼───────────────────────┼──────────────┼──────────────────────────────────────┼ 
```

AFTER:
```
%exo sos acl list bucket-test object-test.pdf
┼───────────────────────┼──────────────┼──────────────────────────────────────┼
│       FILE NAME       │     ACL      │                VALUE                 │
┼───────────────────────┼──────────────┼──────────────────────────────────────┼
│ object-test.pdf       │ Write        │ joh-test@exoscale.ch                 │
│ object-test.pdf       │ Full Control │ joh-test@exoscale.ch                 │
┼───────────────────────┼──────────────┼──────────────────────────────────────┼ 
```

### CODE REVIEW

File: cmd/sos_acl.go [hunk 1]
```diff
@@ -450,14 +450,9 @@ var sosShowACLCmd = &cobra.Command{
...
-
-		for k, v := range objInfo.Metadata {
-			if len(v) > 0 {
-				if isGrantACL(k) {
-					s := getGrantValue(v)
-					table.Append([]string{objInfo.Key, formatGrantKey(k), s})
-				}
+		} else {
+			for _, g := range objInfo.Grant {
+				table.Append([]string{objInfo.Key, formatGrantKey(g.Permission), g.Grantee.DisplayName})
```
Comment (pierre-emmanuelJ) [inline cmd/sos_acl.go]: 👍

File: cmd/sos_acl.go [hunk 2]
```diff
@@ -467,35 +462,23 @@ var sosShowACLCmd = &cobra.Command{
...
+	case k == "WRITE":
 		res = "Write"
-	case k == manualReadACP:
+	case k == "READ_ACP":
 		res = "Read ACP"
-	case k == manualWriteACP:
+	case k == "WRITE_ACP":
 		res = "Write ACP"
-	case k == manualFullControl:
+	case k == "FULL_CONTROL":
```
Comment (pierre-emmanuelJ) [inline cmd/sos_acl.go]: You should Add those value at the top of the file in const. And add a comment to explain, those value come from the S3 response body.
If you do that could you edit this comment also:
https://github.com/exoscale/cli/blob/7d7502b1399731edfd610de6edd1d9be6cab4e66/cmd/sos_acl.go#L24
by explaining, those value come from the S3 response header. 

Thank you great job! 🚀

Comment (falzm) [reply cmd/sos_acl.go]: Good point @pierre-emmanuelJ. I would also recommend that you use this form of `switch` statement:

```go
switch k {
    case SOSACLRead:
        res = "Read"
    case SOSACLWrite:
        res = "Write"
    // ...
}
```

File: cmd/sos_acl.go [hunk 3]
```diff
@@ -21,12 +21,19 @@ const (
...
 
-	//Manual edit ACLs
+	//S3 Grant ACLs response header
 	manualRead        string = "X-Amz-Grant-Read"
 	manualWrite       string = "X-Amz-Grant-Write"
 	manualReadACP     string = "X-Amz-Grant-Read-Acp"
 	manualWriteACP    string = "X-Amz-Grant-Write-Acp"
 	manualFullControl string = "X-Amz-Grant-Full-Control"
+
+	//S3 Grant ACLs response body
```
Comment (falzm) [inline cmd/sos_acl.go]: Each package-exported symbol must be independently described.

**Pull request merged**